events/wc_dark_cult_events.txt

```diff
@@ -1114,6 +1170,245 @@ narrative_event = {
 				character_event = { id = MNM.10030 }
 			}
 			narrative_event = { id = WCDAC.5050 }
+			character_event = { id = WCDAC.5041 days = 7 }
+			character_event = { id = WCDAC.5045 years = 5 }
+		}
+	}
+}
+character_event = {
+	id = WCDAC.5041
```

Comment:
My fellow 4 year old centaur girl got stabbed in the back after visiting public brawls.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79264748-15955300-7ea6-11ea-93b1-d9656b03f099.png)

</details>